AnnoDesigner/Localization/Localization.cs

```diff
@@ -182,7 +182,19 @@ public static void Init(ICommons commons)
                         { "ViewLicenses" , "View open source Licenses" },
                         { "ExternalLinkConfirmationMessage" , "This will open a new tab in your default web browser. Continue?" },
                         { "ExternalLinkMessageTitle" , "Opening an external link" },
-                        { "RecentFiles" , "Recent Files" }
+                        { "RecentFiles" , "Recent Files" },
+                        { "UpdatePrefCurrentVersion" , "Current Version" },
```

Comment:
I feel like this may read better as "Version Information", rather than "Current Version"? Especially as we have the version information for several items below this heading.

Reply:
I will change it tomorrow to "Version Information" (including localizations)